rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRestInterface.java

```diff
@@ -126,7 +126,9 @@ String loginWithCredential(@MultipartForm LoginForm multipart)
     @GZIP
     @Path("monitoring")
     @Produces("application/json")
-    RMInitialState getInitialState(@HeaderParam("sessionid") String sessionId) throws NotConnectedException;
+    RMInitialState getInitialState(@HeaderParam("sessionid") String sessionId,
+            @HeaderParam("latestCounter") @DefaultValue("0") String latestCounterClientAware)
```

Comment:
why a different default value than the other one?

Reply:
It is a bug. It has to be -1.